.github/workflows/ci.yml

```diff
@@ -10,12 +10,5 @@ on:
     branches: [ master ]
 
 jobs:
```

Comment:
is an empty "jobs:" section relevant anymore?

Reply:
It's not empty, there is still the `build` job.